sdk/identity/azure-identity/azure/identity/_internal/msal_credentials.py

```diff
@@ -105,7 +91,8 @@ def __init__(self, client_id, client_credential=None, **kwargs):
         self._cache = kwargs.pop("_cache", None)  # internal, for use in tests
         if not self._cache:
             if kwargs.pop("enable_persistent_cache", False):
-                self._cache = _load_persistent_cache()
+                allow_unencrypted = kwargs.pop("allow_unencrypted_cache", False)
```

Comment:
Is this a public kwarg?

I don't see it in docstring.

Reply:
Yes. Check the diffs for browser.py and device_code.py.

Reply:
Do you mean users are not supposed to use MsalCredential directly?

Reply:
Yes.